### PR TITLE
[117] Do overlay and rotation even on posts without thumbnails/featur…

### PR DESCRIPTION
**Pull request by katelovescode**
…ed images for consistency of look

### CODE REVIEW

File: app/wp-content/themes/kelly/style.css [hunk 1]
```diff
@@ -583,11 +583,11 @@ a:active {
 	background-position: center center;
 	background-size: cover;
 }
-.has-post-thumbnail .entry-title .title-overlay {
+.has-post-thumbnail .entry-title .title-overlay, .entry-title .title-overlay {
```
Comment (dispatchrabbi) [inline app/wp-content/themes/kelly/style.css]: Why not just remove `.has-post-thumbnail` from these two lines and leave them as `.entry-title .title-overlay`?

Comment (katelovescode) [reply app/wp-content/themes/kelly/style.css]: https://media.giphy.com/media/3o85xIO33l7RlmLR4I/giphy.gif

**Comment by dispatchrabbi** [comment]
LGTM! :shipit:

**Pull request merged**